src/Commands/PowerPlatform/Environment/GetPowerPlatformConnectors.cs

```diff
@@ -0,0 +1,65 @@
+﻿using PnP.PowerShell.Commands.Attributes;
+using PnP.PowerShell.Commands.Base;
+using PnP.PowerShell.Commands.Utilities.REST;
+using System;
+using System.Management.Automation;
+using System.Linq;
+using PnP.PowerShell.Commands.Base.PipeBinds;
+using System.Security.Cryptography.X509Certificates;
+
+namespace PnP.PowerShell.Commands.PowerPlatform.Environment
+{
+    [Cmdlet(VerbsCommon.Get, "PnPPowerPlatformConnectors")]
+    public class GetPowerPlatformConnectors : PnPAzureManagementApiCmdlet
+    {
+      
+        [Parameter(Mandatory = false, ValueFromPipeline = true)]
+        public PowerPlatformEnvironmentPipeBind Environment;
+
+        [Parameter(Mandatory = false)]
+        public SwitchParameter AsAdmin;
+
+        [Parameter(Mandatory = false)]
+        public PowerPlatformConnectorPipeBind Identity; 
+
+        protected override void ExecuteCmdlet()
+        {
+            string environmentName = null;
+            if (ParameterSpecified(nameof(Environment)))
+            {
+                environmentName = Environment.GetName();
+                WriteVerbose($"Using environment as provided '{environmentName}'");
+            }
+            else
+            {
+                var environments = GraphHelper.GetResultCollectionAsync<Model.PowerPlatform.Environment.Environment>(Connection, "https://management.azure.com/providers/Microsoft.ProcessSimple/environments?api-version=2016-11-01", AccessToken).GetAwaiter().GetResult();
+                environmentName = environments.FirstOrDefault(e => e.Properties.IsDefault.HasValue && e.Properties.IsDefault == true)?.Name;
+
+                if (string.IsNullOrEmpty(environmentName))
+                {
+                    throw new Exception($"No default environment found, please pass in a specific environment name using the {nameof(Environment)} parameter");
+                }
+
+                WriteVerbose($"Using default environment as retrieved '{environmentName}'");
+            }
+
+            if (ParameterSpecified(nameof(Identity)))
+            {
+                var appName = Identity.GetName();
+
+                WriteVerbose($"Retrieving specific Custom Connector with the provided name '{appName}' within the environment '{environmentName}'");
+
+                var result = GraphHelper.GetAsync<Model.PowerPlatform.Environment.PowerPlatformConnector>(Connection, $"https://api.powerapps.com/providers/Microsoft.PowerApps{(AsAdmin ? "/scopes/admin/environments/" + environmentName : "")}/apis/{appName}?api-version=2016-11-01&$filter=environment eq '{environmentName}' and isCustomApi eq 'True'", AccessToken).GetAwaiter().GetResult();
```

Comment:
Is the API url generic ? I guess it would be different for different clouds like GCC or GCC high ? Can you investigate it , if possible ? 

We have a somewhat similar thing for Graph, because Graph in commercial cloud is `graph.microsoft.com` whereas in GCC it would be `graph.microsoft.us`